README.md

```diff
@@ -3,54 +3,100 @@
+- [Developer Notes](#developer-notes)
 - [Frequent questions or issues](#frequent-questions-or-issues)
 - [Repo structure notes](#repo-structure-notes)
 - [Resources](#resources)
 
 
+
 ## Installation steps
 
+1. Install `python3.11` or greater.
+
 1. If JavaScript tool installs are needed (never used or installed `npm`, `nvm`, `node`):
     1. Here are the recommended [steps to install `nvm`](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating). `nvm` is a node version manager that streamlines installing the other 2.
     - Otherwise (not recommended): [steps to individually install `node` and `npm`](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
 1. Once you have `nvm`, install the latest of both `node` and `npm` with:
-    ```shell
+    ```bash
     nvm install --lts
     ```
-
+1. Ensure docker or podman is installed on your machine.
 1. One package depends on having Chrome (brower) or Chromium (browser driver) installed. **If you have Chrome installed, you can skip this step**. Otherwise, install Chrome or Chromium.
-Here is the link for the [Chromium download](https://www.chromium.org/getting-involved/download-chromium/).
-
-1. Clone this repo and install the requirements:
-    ```shell
-    git clone https://github.com/carterwward/py-allotax.git &&
-    cd py-allotax &&
-    npm install allotaxonometer &&
-    pip3 install pandas pyhtml2pdf selenium==4.25.0
-    ```
+
+## Package Build
+Clone this repo and install the requirements:
+
+```bash
+git clone https://github.com/carterwward/py-allotax.git &&
+cd py-allotax &&
+./scripts/build.sh
+```
+
+You should see a `.whl` file in the newly created `dist` directory.
 
 ## Usage instructions
-*(also see examples.ipynb)*
-1. Verify your data is in the required format (`.json`) by seeing json examples in `convert/`.
-    - See helper functions in `utils` to convert among `csv`, `json`, and `js` formats.
-    - The method `utils.convert_csv_data` exists to convert your data from `.csv` to `.json` if needed--see `examples.ipynb`.
-1. Add your 2 system's files. You need 2 `data.json` files, one for each system.
-1. Run the following to test to verify that the library works. Note `alpha` needs to be passed as a string. If using`infinity`, pass `"inf"`.
-    ```shell
-    cd src &&
-    python3 generate_svg.py convert/boys_2022.json convert/boys_2023.json output_charts/test.pdf "0.17" "Boys 2022" "Boys 2023"
+
+### Scripting
+If working in a python notebook or scripting, you can install the package and use the function directly:
+
+1. Create a new project or directory outside the `py-allotax` repo.
+2. from `py-allotax`
+    ```bash
+    mkdir ../<new repo name> &&
+    cd ../<new repo name>
     ```
-Verify this test against `output_charts/test.pdf`.
-1. If working in a python notebook or scripting, you can import the function and use it directly:
+3. Create a new project environment:
+    -   ```bash
+        python3.11 -m venv .venv
```

Comment:
i'm just gonna say it, some people don't like to use `venv`. I'm fine with it now.